crates/languages/src/python.rs

```diff
@@ -629,14 +629,26 @@ impl ToolchainLister for PythonToolchainProvider {
         let mut toolchains: Vec<_> = toolchains
             .into_iter()
             .filter_map(|toolchain| {
-                let name = if let Some(version) = &toolchain.version {
-                    format!("Python {version} ({:?})", toolchain.kind?)
-                } else {
-                    format!("{:?}", toolchain.kind?)
+                let mut name = String::from("Python");
+                if let Some(ref version) = toolchain.version {
+                    name.push(' ');
+                    name.push_str(&version);
                 }
-                .into();
+
+                let name_and_kind = match (&toolchain.name, &toolchain.kind) {
+                    (Some(name), Some(kind)) => Some(format!("({name}; {:?})", kind)),
```

Comment:
Could we add a helper to get a human readable kind label? Relying on debug impl for pretty printing is far from ideal.